test/e2e-tests.sh

```diff
@@ -33,7 +33,12 @@ fi
 (( !failed )) && downstream_eventing_e2e_tests || failed=7
 
 if [[ $TEST_KNATIVE_KAFKA == true ]]; then
- (( !failed )) && downstream_knative_kafka_e2e_tests || failed=8
+ (( !failed )) && ensure_kafka_no_auth || failed=8
+ (( !failed )) && downstream_knative_kafka_e2e_tests || failed=9
+# (( !failed )) && ensure_kafka_tls_auth || failed=10
+# (( !failed )) && downstream_knative_kafka_e2e_tests || failed=11
+# (( !failed )) && ensure_kafka_sasl_auth || failed=12
+# (( !failed )) && downstream_knative_kafka_e2e_tests || failed=13
```

Comment:
The downstream smoke tests with TLS and SASL is disabled for now since we don't have those changes in the eventing code used in this repo yet

Reply:
Created https://issues.redhat.com/browse/SRVKE-650 to track this